pkg/tablebackup/basebackup.go

```diff
@@ -19,16 +19,20 @@ import (
 	"github.com/ikitiki/logical_backup/pkg/utils"
 )
 
-func (t *TableBackup) RunBasebackup() error {
+// RunBaseBackup produces a 'snapshot' of the table (using COPY).
+// It returns false if the table doesn't exist, otherwise true alongside the error if any
+func (t *TableBackup) RunBasebackup() (bool, error) {
```

Comment:
maybe we should introduce `tableNotFound` error and return it here, instead of `bool` return param.